core/src/test/scala/org/apache/spark/internal/config/ConfigEntrySuite.scala

```diff
@@ -51,6 +52,26 @@ class ConfigEntrySuite extends SparkFunSuite {
     assert(conf.get(dConf) === 20.0)
   }
 
+  test("conf entry: timezone") {
```

Comment:
also move this test to the end, it's weird to put this test in the middle of different config types tests.